File: libs/common/models/provider-website-dto.ts



export class ProviderWebsiteDto {
    public providerWebsiteId: number = 0;
    public websiteUsage: string = '';
    public listSequence: number;
    public websiteUrl: string = '';
    public providerId: number;
    public businessStartDate: Date;
    public businessEndDate: Date;

    constructor(init? : Partial<ProviderWebsiteDto>) {
        
        Object.assign(this, init);
    }
}

File: apps/institution-api/Models/Dto/ProviderWebsiteDto.cs
using System;

namespace ProviderApi.Models.Dto
{
  public class ProviderWebsiteDto
  {
    public int ProviderWebsiteId { get; set; }
    public string WebsiteUsage { get; set; }
    public int? ListSequence { get; set; }
    public string WebsiteUrl { get; set; }
    public int? ProviderId { get; set; }
    public DateTime? BusinessStartDate { get; set; }
    public DateTime? BusinessEndDate { get; set; }
  }
}
